spec/CourseDbSpec.js

```diff
@@ -104,4 +104,25 @@ describe("Course DB", () => {
         expect(result)
             .toBe(true);
     });
+
+    it("shall be able to return all course names", async () => {
+        await CourseDb.createCourse("SER515", 5,
+            "");
+        await CourseDb.createCourse("SER500", 5,
+            "");
+
+        const result = await CourseDb.getCourses();
+        expect(result)
+            .toEqual(["SER515", "SER500"]);
```

Comment:
Do these ever come back in a different order? Like, ["SER500", "SER515"]. This may need to be checked.

Reply:
changed to 2 contains and a length check